src/lib.rs

```diff
@@ -345,6 +345,9 @@ impl MessageWriter {
             .write_all(&*data)
             .await
             .expect("error writing to stdout");
+        // marmarek: while testing I got `error writing to stdout: 0s { code: 32, kind: BrokenPipe,
+        // message: "Broken pipe" }` once, but it still worked after that; maybe this doesn't
+        // expect O_NONBLOCK? but that doesn't match BrokenPipe...
```

Comment:
I’ll have to figure out what is happening here.  Could be (but probably isn’t) a qrexec bug.